ui-frontend/packages/catalog-ui-search/src/main/webapp/react-component/utils/validation/validation.tsx

```diff
@@ -263,6 +270,60 @@ function validateLinePolygon(mode: string, currentValue: string) {
   }
 }
 
+function validateBoundingBox(value: any) {
+  let north
+  let south
+  if (value.dms) {
+    const coordinateNorth = parseDmsCoordinate(value.north)
+    const coordinateSouth = parseDmsCoordinate(value.south)
+    north = coordinateNorth
+      ? dmsCoordinateToDD({
+          ...coordinateNorth,
+          direction: value.dmsNorthDirection,
+        })
+      : null
+    south = coordinateSouth
+      ? dmsCoordinateToDD({
+          ...coordinateSouth,
+          direction: value.dmsSouthDirection,
+        })
+      : null
+  } else if (value.usng) {
+    const upperLeftCoord = converter.USNGtoLL(value.upperLeft, true)
```

Comment:
```
    const upperLeftCoord = converter.USNGtoLL(value.upperLeft, true)
    const lowerRightCoord = converter.USNGtoLL(value.lowerRight, true)
    north = upperLeftCoord.lat
    south = lowerRightCoord.lat
```
✏️  super nit-pick but it might look a shade cleaner like this